packages/tx/src/legacyTransaction.ts

```diff
@@ -382,7 +394,7 @@ export default class Transaction extends BaseTransaction<Transaction> {
     // No unsigned tx and EIP-155 activated and chain ID included
     if (
       v !== undefined &&
-      !v.eqn(0) &&
+      !v.eqn(0) && // TODO fixme -> why overwrite/return default common when `v` is set to 0?
```

Comment:
Ah yes I forgot I wanted to suggest to remove this check. I introduced it a while back but I found no good reason why I introduced it back then (probably some tests failing or something). So yes this can be removed.

```suggestion
```